From bea4236c74d6e45753161d03fdbf93e5a7eb8e00 Mon Sep 17 00:00:00 2001
From: Paolo Valente <paolo.valente () linaro ! org>
Date: Sat, 13 Apr 2019 09:16:26 +0000
Subject: [PATCH] block, Kconfig.iosched: set default value of IOSCHED_BFQ to
 yes

When I added bfq to Kconfig.iosched, I forgot to add the line that
sets to yes the default value of IOSCHED_BFQ, as it is the case for
the other two I/O schedulers. Such a piece of information is used by
popular build systems, when making kernel configs automatically. So,
the lack of this line for bfq makes it hard and confusing to have bfq
in the kernel generated by these systems.

This commit adds this missing line.

Reported-by: Anders Roxell <anders.roxell@linaro.org>
Signed-off-by: Paolo Valente <paolo.valente@linaro.org>
---
 block/Kconfig.iosched | 1 +
 1 file changed, 1 insertion(+)

diff --git a/block/Kconfig.iosched b/block/Kconfig.iosched
index 4626b88..01d05ab 100644
--- a/block/Kconfig.iosched
+++ b/block/Kconfig.iosched
@@ -20,6 +20,7 @@ config MQ_IOSCHED_KYBER
 
 config IOSCHED_BFQ
 	tristate "BFQ I/O scheduler"
+	default y
 	---help---
 	BFQ I/O scheduler for BLK-MQ. BFQ distributes the bandwidth of
 	of the device among all processes according to their weights,
-- 
2.22.0.214.g8dca754b1e

